src/compile/data/geopoint.ts

```diff
@@ -0,0 +1,64 @@
+import {isArray} from 'vega-util';
+import {Channel, SingleDefChannel, X, X2, Y, Y2} from '../../channel';
+import {channel} from '../../index';
+import {LATITUDE, LONGITUDE} from '../../type';
+import {contains, duplicate, keys} from '../../util';
+import {VgGeoPointTransform} from '../../vega.schema';
+import {ModelWithField} from '../model';
+import {DataFlowNode} from './dataflow';
+
+export class GeoPointNode extends DataFlowNode {
+  public clone() {
+    return new GeoPointNode(this.projection, duplicate(this.fields), duplicate(this.as));
+  }
+
+  constructor(private projection: string, private fields: string[], private as: string[]) {
+    super();
+  }
+
+  public static makeAll(model: ModelWithField): GeoPointNode[] {
```

Comment:
this is different. If they're using a line, I can pull out multiple GeoPoint transforms from one model. Is this a good way of doing that @domoritz?